### PR TITLE
extra-headings-support - Adding support for other blockmode headings

### PR DESCRIPTION
**Pull request by mitermayer**
- adding support for other blockmode headings

### CODE REVIEW

**Comment by mitermayer** [comment]
this solves https://github.com/facebook/draft-js/issues/95

File: website/src/draft-js/index.js [hunk 1]
```diff
@@ -145,6 +145,9 @@ class StyleButton extends React.Component {
 const BLOCK_TYPES = [
   {label: 'H1', style: 'header-one'},
   {label: 'H2', style: 'header-two'},
+  {label: 'H3', style: 'header-three'},
```
Comment (mitermayer) [inline website/src/draft-js/index.js]: Also added to the website demo

File: src/component/utils/getElementForBlockType.js [hunk 2]
```diff
@@ -17,9 +17,15 @@ import type {DraftBlockType} from 'DraftBlockType';
 function getElementForBlockType(blockType: DraftBlockType): string {
   switch (blockType) {
     case 'header-one':
-      return 'h2';
```
Comment (mitermayer) [inline src/component/utils/getElementForBlockType.js]: I believe this was a typo ? so I updated the header-one to return h1. Is that a problem or was that the intendend behaviour ? If so I can amend the PR ?

Comment (hellendag) [inline src/component/utils/getElementForBlockType.js]: This was intended, but is a decision from long ago. I'll check to see if we can go ahead and change it. Your change is surely more sane for a public repo. :)

**Comment by hellendag** [comment]
@facebook-github-bot import

**Comment by hellendag** [comment]
I'm going to go ahead and add `header-six` to the imported diff to round it out. :)

File: src/model/constants/DraftBlockType.js [hunk 3]
```diff
@@ -20,6 +20,9 @@ export type DraftBlockType = (
   'paragraph' |
   'header-one' |
   'header-two' |
+  'header-three' |
+  'header-four' |
+  'header-five' |
```
Comment (sstur) [inline src/model/constants/DraftBlockType.js]: Why stop at five? It just seems like an arbitrary choice between 1 and 6. Maybe a better choice than 2, but interested to know why you chose 5.

Comment (hellendag) [inline src/model/constants/DraftBlockType.js]: imgres

Comment (sstur) [inline src/model/constants/DraftBlockType.js]: Hahaha, by that logic, we should include h7!

**Pull request closed without merging**